pkgs/development/interpreters/php/default.nix

```diff
@@ -136,6 +136,7 @@ let
       ++ optionals mysqliSupport [
         "--with-mysqli=${if mysqlndSupport then "mysqlnd" else "${mysql.connector-c}/bin/mysql_config"}"
       ]
+      ++ optional ( pdo_mysqlSupport || mysqlSupport || mysqliSupport ) "--with-mysql-sock=/run/mysqld/mysqld.sock"
```

Comment:
Nitpick: Can this be always set independent if mysql modules are enabled or not?

Reply:
Most configure flags are set conditionally on the *Support arguments - I didn't want to clutter the configure output with flags that aren't used.